### PR TITLE
Update Python Requirements

### PR DESCRIPTION
**Pull request by itdependsnetworks**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -26,7 +27,7 @@ include = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.6.2"
+python = "^3.6"
```
Comment (jeffkala) [inline pyproject.toml:30]: Does this no longer break `black`?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -137,7 +137,7 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        python-version: ["3.6", "3.7", "3.8", "3.9"]
+        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10"]
```
Comment (jeffkala) [inline .github/workflows/ci.yml:140]: 3.10 is interesting, trying to find the article but read somewhere that 3.10 and `black` don't interoperate accurately right now.

Comment (itdependsnetworks) [reply .github/workflows/ci.yml:140]: tests passed :)

File: pyproject.toml (hunk 1 above)
Comment (itdependsnetworks) [reply pyproject.toml:30]: Black is a development dependency, even is this is the case, it should not matter.

**Pull request merged**